client/src/components/DataFiles/DataFilesBreadcrumbs/DataFilesBreadcrumbs.jsx

```diff
@@ -81,7 +82,7 @@ const RootProjectsLink = ({ api, section, operation, label }) => {
         type: 'DATA_FILES_SET_MODAL_PROPS',
         payload: {
           operation,
-          props: { showProjects: true },
+          props: { ...modalProps, showProjects: true },
```

Comment:
`...modalProps` holds values such as `canMakePublic`, which if overridden would remove `Public Data` from the dropdown